### DISCUSSION TITLE
kobweb run command issue

### DISCUSSION THREAD

**Discussion by shaxzodshox** [varabyte/kobweb#383] [Q&A]
Hello everyone. I am trying to run my project by entering kobweb run on my Android Studio IDE terminal. I am entering into site module before running. But it is still giving me the following error:

"This command must be called in a Kobweb application module."

Am I doing something wrong? 

Thank you for your response.

**Comment by bitspittle** [comment]
A Kobweb project is reallly a normal kotlin project folder which has a `.kobweb` folder in it, which should furthermore have a `conf.yaml` file in that. If you try to run `kobweb run` in a folder where `.kobweb/conf.yaml` isn't accessible, then you'll get that error.

Did you try to create your Kobweb project by hand? Or did you start with something like `kobweb create app` first?

This section in the README might also help: https://github.com/varabyte/kobweb#adding-kobweb-to-an-existing-project

---

Finally, you can run these steps as an easy way to confirm that `kobweb run` works for you in a different project:

```bash
# in a temp folder somewhere
$ kobweb create app
# accept all defaults
$ cd app/site
$ kobweb run
# at this point, if it's working, feel free to quit the running server and
# delete the project.
```

**Reply by shaxzodshox** [reply]
Hi bitspittle!

Thank you for your response. I created using the terminal by writing `kobweb create`. Project created successfully and said that I can start it using `kobweb run` command. However, the same issue is happened. ( There is `.kobweb`folder with `conf.yaml` file in my project's structure.
